src/js/jquery.swipebox.js

```diff
@@ -627,6 +627,13 @@
 					slider.animate( { left : ( -index*100 )+'%' } );
 				}
 
+				if ( index === 0 ) {
+					$( '#swipebox-prev' ).addClass( 'disabled' );
+				} else if ( index === elements.length - 1 && plugin.settings.loopAtEnd !== true ) {
+					$( '#swipebox-next' ).addClass( 'disabled' );
+					$( '#swipebox-slider .slide' ).eq( index ).addClass( 'Last_Slide' );
```

Comment:
Why have we gone with `Last_Slide` as the class name?

Reply:
Because I had to change the swipebox code as well so it applies it to every last slide

Reply:
Sorry, I meany why the format `Last_Slide`? I've not looked around Swipebox much but typically we'd do `last-slide`. If it's fitting in with what's there, no problem.

Reply:
Looks like just my bad habits thats all